src/html.ml

```diff
@@ -10,52 +10,48 @@ type printer =
     code_block: printer     -> Buffer.t -> Code_block.t              -> unit;
     thematic_break: printer -> Buffer.t                              -> unit;
     html_block: printer     -> Buffer.t -> string                    -> unit;
-    heading: printer        -> Buffer.t -> int -> inline             -> unit;
+    heading: printer        -> Buffer.t -> inline Heading.t          -> unit;
     inline: printer         -> Buffer.t -> inline                    -> unit;
     concat: printer         -> Buffer.t -> inline list               -> unit;
     text: printer           -> Buffer.t -> string                    -> unit;
     emph: printer           -> Buffer.t -> inline Emph.t             -> unit;
-    code: printer           -> Buffer.t -> int -> string             -> unit;
+    code: printer           -> Buffer.t -> Code.t                    -> unit;
     hard_break: printer     -> Buffer.t                              -> unit;
     soft_break: printer     -> Buffer.t                              -> unit;
     html: printer           -> Buffer.t -> string                    -> unit;
     link: printer           -> Buffer.t -> inline Link.t             -> unit;
     ref: printer            -> Buffer.t -> inline Ref.t              -> unit;
   }
 
-let _id_of_string ids s =
-  let n = String.length s in
-  let out = Buffer.create 0 in
-  (* Put [s] into [b], replacing non-alphanumeric characters with dashes. *)
-  let rec loop started i =
-    if i = n then ()
+let id_of_string s =
```

Comment:
Sorry, I can't see where this is used; and why is `s` of type `bytes` and not `string` ?

Reply:
This is actually a relic of before my work on omd, I may have used it for something and forgot to delete it afterward